src/components/Common/SettingsPage/SetingsPage.jsx

```diff
@@ -0,0 +1,41 @@
+import { Offcanvas } from 'react-bootstrap';
+import { SettingsContext } from '../../../Hooks/useSettings';
+import { Button } from '../../Buttons/Button/Button';
+import style from './SetingsPage.module.css';
+
+export function SettingsPge() {
+  const { body } = document;
+  const setLightTheme = () => {
+    body.setAttribute('data-theme', 'light');
+  };
+  const setDarkTheme = () => {
+    body.setAttribute('data-theme', 'dark');
+  };
+  const setDarkDimmedTheme = () => {
+    body.setAttribute('data-theme', 'dark-dimmed');
+  };
```

Comment:
These methods should be in ThemeContext and create one `toggleTheme(theme)` method.

Reply:
```
  toggleTheme(theme) {
    this.setState((prevState) => ({ theme: { ...prevState.theme, theme } }));
  }
```